### DISCUSSION TITLE
Standard Formatter

### DISCUSSION THREAD

**Discussion by Prajwalprakash3722** [IEEE-RVCE/GateKeeper#7] [Ideas]
Let's all use a standard formatter which can help to code consistent code, 

Suggestions: 
- [spotless](https://github.com/diffplug/spotless)

**Comment by chrisvrose** [comment]
`spotless` looks nice.
Additionally, if we're using Intellij/VSCode, the default formatter works rather well.
However, it would be beneficial to format before pushing code.

**Reply by Prajwalprakash3722** [reply]
alright, maybe a pre-hook makes sense in this situation?
